src/main/java/org/mule/tools/devkit/sonar/utils/PomUtils.java

```diff
@@ -54,6 +62,32 @@ public static boolean isDevKitConnector(MavenProject mavenProject) {
         return parent != null && parent.getGroupId().equals(DEVKIT_GROUP_ID) && (parent.getArtifactId().equals(DEVKIT_ARTIFACT_ID) || parent.getArtifactId().equals(CERTIFIED_DEVKIT_ARTIFACT_ID));
     }
 
+    public static VersionUtils getCurrentDevkitVersion(String devkitVersion){
+        return new VersionUtils(devkitVersion);
+    }
+
+    public static VersionUtils getLatestDevkitVersion() {
+        VersionUtils latestVersion = null;
```

Comment:
If set to NULL, whenever an exception is catched, the rule will throw a NullPointerException when calling `latestVersion.compareTo()`.